qiskit/synthesis/evolution/__init__.py

```diff
@@ -12,6 +12,9 @@
 
 """Synthesis for operator evolution gates."""
 
+# pylint: disable=unused-import
+from qiskit._accelerate.synthesis.evolution import pauli_network_synthesis
```

Comment:
The classes in here perform the full synthesis of a `PauliEvolutionGate` to a `QuantumCircuit`,  which this function does not. Would it make more sense to expose a similar wrapper here (basically what the plugin is doing) or just to remove this function as it is more a utility for the plugin?

Reply:
I agree. To be consistent with the other synthesis methods, I have wrapped this into a function `synth_pauli_network_rustiq`.